CONFIGURATION.md

```diff
@@ -84,8 +84,10 @@ The other placeholders are specified separately.
   # HTTP proxy server to use to connect to the targets.
   [ proxy_url: <string> ]
 
-  # The preferred IP protocol of the HTTP probe (ip4, ip6).
-  [ preferred_ip_protocol: <string> | default = "ip6" ]
+  # The IP protocol of the HTTP probe (ip4, ip6).
+  [ preferred_ip_protocol: <string> | default = "ip6" *DEPRECATED* ]
```

Comment:
I don't see why we're deprecating this just to change its name, this would cause unnecessary work for all our existing users.

Reply:
I never liked the name or behavior of this code. With the new code, it's nolonger exactly "preferred", it's just specifying a specific protocol, and having a fallback or not.

I do not want to go in circles on this review.

Reply:
It's still preferred, and we've already broken this config option before. If we're going to break it again I want to to be due a complete change in how it works, not a minor name change.

Reply:
It is not, the new behavior does not represent preferred anymore. With preferred, fallback was always an option. Now fallback is optional, and this necessitates a name change.

Reply:
I don't think that adding a new option necessitates breaking existing users.